### ISSUE TITLE
Combine adjacent image and text links for the same resource

### ISSUE THREAD

**Issue by hostle83** [DSpace/dspace-angular#3070]
**Describe the bug**
Adjacent text and images which navigate to the same destination should be presented and announced as one link. A description of this WCAG standard can be found here: https://www.w3.org/WAI/WCAG20/Techniques/html/H2 

**To Reproduce**
1. On the sandbox or demo site, go to any entry that displays a thumbnail next to the textual description. 
2. If the thumbnail has a link that takes the user to the same place as the text link, these two links should be combined semantically in the html code. E.g.:

`<a href="products.html">
   <img src="icon.gif" alt="">
   Products page
 </a>
`

**Expected behavior**
Expect that screen readers and other assistive technology does not encounter the same links a number of times, but rather just once (see code above)

**Comment by atarix83** [comment]
@hostle83 

I can't figure out where this issue is detected. I thought in the search page where we have thumbnails and item title with the same url, but using the axe devtools  the problem is not reported
![immagine](https://github.com/DSpace/dspace-angular/assets/2486489/801e15ad-c38d-4e36-a391-1d24f5fed6c6)

Can i ask where did you find the problem? thanks

**Comment by tdonohue** [comment]
@atarix83 : I'm able to detect this issue using [WAVE](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US), but **not** using Axe DevTools.  I think it's worth noting that different scanners will find different issues sometimes.

If I run WAVE scanner on that Search Results page (where thumbnails are shown), I see a large number of "Alerts". In WAVE, an "Alert" is an accessibility warning, but *not* an accessibility error. 

Some of these alerts appear next to the Title link of every Item in the search results and say: `Redundant Link: Adjacent links go to the same URL`.  

The full alert details from WAVE say:
> **What It Means**
>Adjacent links go to the same URL.
>
>**Why It Matters**
>When adjacent links go to the same location (such as a linked product image and an adjacent linked product name that go to the same product page) this results in additional navigation and repetition for keyboard and screen reader users.
>
>**How to Fix It**
>If possible, combine the redundant links into one link and remove any redundant text or alternative text (for example, if a product image and product name are in the same link, the image can usually be given alt="").
>
>**Standards and Guidelines**
[2.4.4 Link Purpose (In Context) (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.4)

So, I think this ticket is describing an accessibility issue, but it's more of a low to medium priority issue.  It's not an accessibility error, but instead is describing a change that would improve accessibility of these pages.

**Comment by atarix83** [comment]
@tdonohue 

for what i can see. considering how the components are structured, if we really want to resolve this issue the easiest solution is to remove the link from the thumbnail and leave only the title as a link.

**Comment by tdonohue** [comment]
@atarix83 : Good point.  I think the question then is which behavior we feel is better.  As I noted, WAVE reports this issue as a **warning** and not an **error**.  So, it's not deemed high priority by WAVE and it undetected by Axe DevTools.

At this point, I think this needs more discussion & might be flagged as lower priority.  There are two clear options:
1. Keep the link on the thumbnail (which may provide better usability but results in an accessibility warning)
2. Remove the link on the thumbnail (which fixes the accessibility warning, but may be seen as detrimental to usability for users who want to be able to click on that image to go to the Item page)

I don't have a strong opinion at this time, but would be interested in what others think.

**Comment by atarix83** [comment]
In my opinion if it's only reported as a warning i prefer the option 1 where usability is kept as is

**Comment by paulo-graca** [comment]
Not a solution, just a design and relevancy choice. We opted to remove thumbnails from listings also adding a call for action button to see more defails:
![Image](https://github.com/user-attachments/assets/1e1d42d8-bc4a-4878-927d-ee386e51bfb7)

Leaving thumbnails for the detail/item page:
![Image](https://github.com/user-attachments/assets/dd223640-5d01-4e80-883b-3969f08a3f5d)

Meaning, we don't have this accessibility problem (having Img and Text on the same <a> element).

**Comment by hostle83** [comment]
Personally, I like the design offered by @paulo-graca very much. It circumvents the need to apply this standard to the layout. 

If we keep the current layout, I would be interested if we have any data about what users click on to access the item - the thumbnail or the text? Like a heatmap analysis, for example?

**Comment by tdonohue** [comment]
In today's Developers Meeting, we had also briefly discussed finding a way to make the thumbnail "only clickable by a mouse".  This would mean potentially using Javascript instead of an HTML `<a href>` tag for the thumbnail.   This would retain the current behavior, but it's less ideal to build this into Javascript instead of using HTML.

Nonetheless, as we decided, this seems "low priority".  We don't have a clear easy solution (though all the brainstorms here are wonderful) because the structure of the search results makes it very difficult to "group" the thumbnail and the title in the same `<a href>` tag (those two are in different columns, and only "visually" next to one another).

I'm also not against @paulo-graca 's approach, but that may require running past DCAT (or similar). I'm not sure how important it is for others to see the thumbnails as part of the search results versus only seeing them after clicking on an item in the search results.
